src/_support/Page/Acceptance/Administrator/UserNotesFormPage.php

```diff
@@ -0,0 +1,69 @@
+<?php
+/**
+ * @package     Joomla.Test
+ * @subpackage  AcceptanceTester.Page
+ *
+ * @copyright   Copyright (C) 2005 - 2017 Open Source Matters, Inc. All rights reserved.
+ * @license     GNU General Public License version 2 or later; see LICENSE.txt
+ */
+namespace Page\Acceptance\Administrator;
+
+/**
+ * Acceptance Page object class to user form page.
+ *
+ * @category  Users
+ * @package   Page\Acceptance\Administrator
+ * @copyright 2005 - 2018 Open Source Matters, Inc. All rights reserved.
+ * @license   GNU General Public License version 2 or later; see LICENSE.txt
+ * @since     4.0.0
+ */
+class UserNotesFormPage
+{
+	/**
+	 * Subject
+	 *
+	 * @var    array
+	 * @since  4.0.0
+	 */
+	public static $subject = '#jform_subject';
+
+	/**
+	 * Select User Button
+	 *
+	 * @var    array
+	 * @since  4.0.0
+	 */
+	public static $selectUserButton = ['xpath' => '//a[@title="Select User"]'];
+
+	/**
+	 * Select category
+	 *
+	 * @var    array
+	 * @since  4.0.0
+	 */
+	public static $selectCategory = '#jform_catid_select';
```

Comment:
Not an array

Reply:
@dneukirchen you mean `@var` should be changed to string, because I asked him to change these to String, to ensure compatibility with codeception 2.4.x

Reply:
yes.

Reply:
added string now